console-subscriber/src/builder.rs

```diff
@@ -456,6 +459,56 @@ impl Builder {
     }
 }
 
+/// Specifies the address on which a [`Server`] should listen.
+///
+/// [`Server`]: crate::Server
+#[derive(Clone, Debug)]
+pub enum ServerAddr {
+    /// A TCP address.
+    Tcp(SocketAddr),
+    /// A Unix socket address.
+    Unix(PathBuf),
```

Comment:
This should probably have `#[cfg(unix)]` so that the API is only available when building for Unix.
```suggestion
    /// A Unix socket address.
    #[cfg(unix)]
    Unix(PathBuf),
```

Reply:
Done.